src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,59 @@
+/* Background bleed */
+
+.bleedWrapper {
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  justify-content: center;
+  padding: 35px;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  display: flex;
+}
+
+.title {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  margin-right: 97px;
+  width: 263px;
```

Comment:
Please don't set a width on the title. The width should be allowed to adjust according to the amount of text in it.

Reply:
ok